examples/l3out_ospf_external_policy/main.tf

```diff
@@ -19,9 +19,9 @@ resource "aci_l3out_ospf_external_policy" "example" {
   l3_outside_dn     = aci_l3_outside.example.id
   annotation        = "example"
   area_cost         = "1"
-  area_ctrl         = "redistribute,summary"
```

Comment:
missing summary?

Reply:
area_ctrl is actually a list that takes only one argument

Reply:
done